tools/run.py

```diff
@@ -115,6 +118,46 @@ def run(args, use_gdb, trace, trace_output, **kwargs):
     else:
         cmd = []
 
+    # Update the config files if requested to support running multiple robots
+    update_configs(webots_port, player_id, team_id)
+
     # Run the command
     pty = WrapPty()
     exit(pty.spawn(cmd + args, env))
+
+# Update config
+def update_config(yaml_file, key, value):
+    # Try to open the file, if it fails don't do anything
+    try:
+        with open(yaml_file, "r") as file:
+            config = yaml.load(file, Loader=yaml.FullLoader)
+    except FileNotFoundError:
+        print(f"config file not found: {yaml_file}, module not built.")
+        return
+
+    # Update the config
+    config[key] = value
+
+    # Write the updated config back to the file
+    with open(yaml_file, "w") as file:
+        yaml.dump(config, file)
+
+def update_configs(webots_port, player_id, team_id):
+    # Change into the config directory
+    os.chdir("./config")
```

Comment:
nit, but is changing directory really needed? Could just add `config/` to start of yaml files right?

Reply:
I would generally avoid doing a change of directory in a script